app/src/main/java/uk/org/openseizuredetector/MainActivity.java

```diff
@@ -246,6 +232,53 @@ public boolean onCreateOptionsMenu(Menu menu) {
         Log.i(TAG, "onCreateOptionsMenu()");
         getMenuInflater().inflate(R.menu.main_activity_actions, menu);
         MenuCompat.setGroupDividerEnabled(menu, true);
+
+
+
+        /*
+        *   @Override
+            public void onViewCreated(View view, Bundle savedInstanceState) {
+                super.onViewCreated(view, savedInstanceState);
+                setHasOptionsMenu(true);
+            }
+            *
+            * // Initialise the User Interface
+        setContentView(R.layout.main);
+        //getWindow().getDecorView().setBackgroundColor(okColour);
+        // Setting in Aroon version of menuKeyFieldFiddle:
+        //first create config as ViewConfiguration and get
+        // ViewConfiguration of this.
+        // than create Field menuKeyField with value null.
+        // if Version.SDK_INT < Build.VERSION_CODES.ICE_CREAM_SANDWICH
+        // run fiddle else check if config.hasPermanentMenuKey() returns inverted false
+        /* Force display of overflow menu - from stackoverflow
+         * "how to force use of..."
+         *--/
+        try {
+            Log.v(TAG, "trying menubar fiddle...");
+            ViewConfiguration config = ViewConfiguration.get(this);
+            //TODO: After API>33 remove next lines and strip until
+            Field menuKeyField = null;
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.ICE_CREAM_SANDWICH) {
+                menuKeyField =
+                        ViewConfiguration.class.getDeclaredField("sHasPermanentMenuKey");
+            } // TODO: here before ! config. start new if( before.
+            if ((menuKeyField != null) ) {
+                Log.v(TAG, "menuKeyField is not null - configuring....");
+                menuKeyField.setAccessible(true);
+                menuKeyField.setBoolean(config, false);
+            } else if (config.hasPermanentMenuKey()) {
+                Log.v(TAG, "menuKeyField is null or config.hasPermanentMenuKey() is true");
+                menuKeyField = getV;
+                        .setAccessible(false);
+                menuKeyField.setBoolean(config, true);
+            }
+        } catch (Exception e) {
+            Log.v(TAG, "menubar fiddle exception: " + e.toString());
+        }
+        */
+
```

Comment:
It is better to undo here: fiddle only if below SDK 29.
If above, unnecessary: 3 dotted is overflow and hardware key is not there.